waltz-web/src/main/java/com/khartec/waltz/web/endpoints/api/SurveyQuestionEndpoint.java

```diff
@@ -74,6 +102,31 @@ public void register() {
     }
 
 
+    private SurveyQuestionInfo mkQuestionInfo(SurveyQuestion question) {
+        ImmutableSurveyQuestionInfo.Builder builder = ImmutableSurveyQuestionInfo.builder();
+
+        if (question.fieldType() == SurveyQuestionFieldType.DROPDOWN) {
+            builder.dropdownEntries(surveyQuestionDropdownEntryService.findForQuestion(question.id().get()));
+        }
+        return builder
+                .question(question)
+                .build();
+    }
+
+
+    private boolean mayBeSaveDropdownEntries(long questionId, SurveyQuestionInfo questionInfo) {
+        checkNotNull(questionInfo.question(), "questionInfo.question() cannot be null");
+
+        if (questionInfo.question().fieldType() == SurveyQuestionFieldType.DROPDOWN) {
+            return surveyQuestionDropdownEntryService.saveEntries(
+                    questionId,
+                    questionInfo.dropdownEntries().orElse(null));
```

Comment:
In fact looking back at the receiver - isn't this guaranteed to fail the check on the service ?